README.md

```diff
@@ -39,55 +74,250 @@ tools (such as `ceph-installer` and `gdeploy`):
+    For **production deployment**, run the full check:
+
+    ```
+    $ ansible-playbook -i inventory_file prechecks.yml
+    ```
+
+    While for **proof of concept deployments**, you can avoid checking of
+    stringent production requirements using `production` tag:
+
+    ```
+    $ ansible-playbook -i inventory_file prechecks.yml --skip-tags "production"
+    ```
+
+    If you are not sure why a particular check is there or what is checked
+    exactly, open the playbook file and see comments and/or implementation of
+    the check.
+
+8)  Then we are ready to run ansible to install Tendrl:
+
+    ```
+    $ ansible-playbook -i inventory_file site.yml
+    ```
+
+    Assuming we have deployed ssh keys on the machines and have Gluster
+    trusted storage pool already installed and running there.
+
+9)  Log in to your tendrl server at ``http://tendrl.example.com`` (hostname
+    of Tendrl server as specified in the inventory file in step #2)  with
+    username ``admin`` and default password ``adminuser``.
+
+    Note that `tendrl-server` role includes setup of admin user account for
+    Tendrl (usable with both api and web interface), and that default
+    password is ``adminuser``. Moreover the admin password is also
+    stored on *Tendrl Server* machine in `/root/password` file (this feature of
+    tendrl-ansible is based
+    on [TEN-257](https://tendrl.atlassian.net/browse/TEN-257)).
+
+
+## How do I expand cluster with tendrl-ansible?
+
+See [Tendrl wiki](https://github.com/Tendrl/documentation/wiki) for full
+details of cluster expansion procedure. This section contains only brief
+overview of the expand operation for you to understand how tendrl-ansible
+fits into Tendrl cluster expand operation.
+
+1)  First of all, you need to install operating system and Gluster on new
+    servers(s) and add them into existing cluster (aka [Gluster *Trusted Storage
+    Pool*](http://docs.gluster.org/en/latest/Administrator%20Guide/Storage%20Pools/))
+    via peer probe and add bricks on new server(s) into existing gluster
+    volume(s) based on your needs.
+
+2)  When Gluster is aware of new servers (you see them in output of `gluster
+    pool list` command), you add the new servers into ansible inventory file
+    (into group `gluster-servers`) which you used during installation of
+    Tendrl.
+
+    Note that it's important to **add new servers into the same inventory file
+    as was used during installation**, because you need to ensure that you are
+    using the same set of ansible variables.
+
+3)  Then, you rerun ansible playbook in the same way as done during Tendrl
+    installation:
+
+    ```
+    $ ansible-playbook -i inventory_file site.yml
+    ```
+
+    During this run, ansible should report "ok" status for
+    already existing machines, while reporting "changed" status for the new
+    machines you just added.
+
+4)  Now, you should be able to see new servers in Tendrl web ui (see Tendrl
+    documentation for details).
```

Comment:
I would add a mention about a notification that should be present after expand is finished: https://github.com/Tendrl/specifications/pull/254/files#diff-8e71bd954ebddb4a0ef4fd3c413d9f04R90

Reply:
I will add there more details, but when we have this resolved https://github.com/Tendrl/commons/issues/849